examples/meta/src/regression/poisson_regression.sg.in

```diff
@@ -0,0 +1,33 @@
+File f_feats_train = read_csv("@SHOGUN_DATA@/regression_1d_linear_features_train.dat")
```

Comment:
+1 for this meta example. I am not so sure it will give meaningful results with these datasets... we might need to add one to run it correctly.

Reply:
Oh okay, should I generate it using pyGLMnet? using the simulate_glm method like I did for the unit test?

Reply:
Yes, that would be better. It shouldn't be too big. Take a look at https://github.com/shogun-toolbox/shogun-data/tree/master/toy for some examples. Once done,  you would need to open a PR also in shogun-data to add this new data file and follow the same procedure as you usually do to add a meta-example.

Reply:
Should I work on this right now? Or a separate PR to add a meta example for GLM after merging this?

Reply:
Datasets added.